### DISCUSSION TITLE
Github interaction not display on github page, instead ONLY posted in my email

### DISCUSSION THREAD

**Discussion by GlutenFreeVapes** [community/community#84671] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

TLDR: Mirror email notifications on the website, don't post links in emails.

Hello, I have made an issue which received a reply.
I received a message about it on my email.
I logged in on github.com and I do not see the message anywhere here.

Using email is not secure, prone to phising and a bad practice.
Please put notifications on the github website and NEVER put any links in email.

(I searched for similar discussions and NO, the HOME page and feed do not display the notification I received in email)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @GlutenFreeVapes! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
